CONTRIBUTING.md

```diff
@@ -1,134 +1,220 @@
-# How to Contribute to Spectrum Web Components
+Welcome! We’re excited you’re interested in improving Spectrum Web Components. Whether you’re reporting bugs, adding new features, writing documentation, or helping other users, your contributions make this project better for everyone.
 
-First off, thanks for taking the time to contribute!
+Here you’ll find a broad overview of how you can get involved. Please read through these guidelines to help keep the contribution process smooth and to ensure we’re all on the same page.
 
-The following are a set of guidelines for contributing to Spectrum Web Components, which is hosted on the [Adobe Spectrum Web Components repository](https://github.com/adobe/spectrum-web-components). These are guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request if the need arises while contributing to the library.
+-   [1. Community \& Support](#1-community--support)
+    -   [1.1. External Contributors](#11-external-contributors)
+    -   [1.2. Internal Contributors](#12-internal-contributors)
+-   [2. Ways to Contribute](#2-ways-to-contribute)
+-   [3. Contributor License Agreement](#3-contributor-license-agreement)
+-   [4. Code of Conduct](#4-code-of-conduct)
+-   [5. Using the Issue Tracker](#5-using-the-issue-tracker)
+    -   [5.1. Before Submitting A Bug Report](#51-before-submitting-a-bug-report)
+-   [6. Bug Reports](#6-bug-reports)
+    -   [6.1. Issue Severity Classification](#61-issue-severity-classification)
+-   [7. Feature Requests/New Component](#7-feature-requestsnew-component)
+-   [8. Pull Requests](#8-pull-requests)
+-   [9. Branches](#9-branches)
+-   [10. Developing Locally](#10-developing-locally)
+-   [11. Testing](#11-testing)
+-   [12. Documentation](#12-documentation)
+-   [13. Best Practices \& Guidelines](#13-best-practices--guidelines)
+    -   [13.1. Code Formatting](#131-code-formatting)
+    -   [13.2. Accessibility](#132-accessibility)
+    -   [13.3. Commit Guidelines](#133-commit-guidelines)
+-   [14. Thank You](#14-thank-you)
 
-## Code Of Conduct
+---
 
-This project adheres to the Adobe [code of conduct](CODE_OF_CONDUCT.md). By participating,
-you are expected to uphold this code. Please report unacceptable behavior to
-[Grp-opensourceoffice@adobe.com](mailto:Grp-opensourceoffice@adobe.com).
+# 1. Community & Support
 
-## TL;DR
+A fantastic first step to contributing is joining our community discussions. This is where you can:
```

Comment:
this is definitely something we should discuss. i know in the past we used discussions but we can also remove this until we are ready to commit.

Reply:
Yeah, I think actually ppl opening an issue is better??? Since we no longer use JUST GH issues for triaging

Reply:
want to take a stab an language around this?

Reply:
Agree with najika, even if we route them to start a discussion, users feel comfortable to discuss in the channel itself and its understandable

Reply:
i was trying to keep the language "open source" focused but we can separate our users concerns of slack for internal users versus github issues for external users. 

Feel free to make a code suggestions to improve this section